File: PDP/DrivingSchoolWebAPI3/DrivingSchoolAPIModels/DbModels/Instructor.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Drawing;

namespace DrivingSchoolAPIModels
{
    /// <summary>
    /// Инструктор
    /// </summary>
    public class Instructor : IPerson
    {
        public int InstructorId { get; set; }
        public string UserId { get; set; }
        [ForeignKey(nameof(UserId))]
        public ApplicationUser User { get; set; }
        [NotMapped]
        public string FirstName { get => User.FirstName; set => User.FirstName = value; }
        [NotMapped]
        public string LastName { get => User.LastName; set => User.LastName = value; }
        [NotMapped]
        public string? Patronymic { get => User.Patronymic; set => User.Patronymic = value; }
        [NotMapped]
        public byte[] ProfilePictureBytes { get => User.ProfilePictureBytes; set => User.ProfilePictureBytes = value; }
        [NotMapped]
        public Image ProfilePicture { get => User.ProfilePicture; set => User.ProfilePicture = value; }

        public override string ToString()
        {
            return $"{LastName} {FirstName}{(Patronymic!=null ? ' '+Patronymic:"")}";
        }
    }
}

File: PDP/DrivingSchoolAppAndroidClient/app/src/main/java/com/example/drivingschoolappandroidclient/api/models/Instructor.kt
package com.example.drivingschoolappandroidclient.api.models

import android.graphics.Bitmap

open class Instructor (
    var instructorId: Int,
    var userId: String,
    var user: ApplicationUser,

): IPerson{

    override var firstName
        get() = user.firstName
        set(value) {user.firstName = value }
    override var lastName: String
        get() = user.lastName
        set(value) {user.lastName = value}
    override var patronymic: String?
        get() = user.patronymic
        set(value) {user.patronymic = value}
    override var profilePictureBytes: ByteArray?
        get() = user.profilePictureBytes
        set(value) {user.profilePictureBytes = value}
    override var profilePicture: Bitmap?
        get() = user.profilePicture
        set(value) {user.profilePicture = value}

    override fun toString() = "$lastName ${firstName}${patronymic?.let { " $patronymic" } } - №$instructorId"
}
